package.json

```diff
@@ -360,6 +360,7 @@
     "embedding-sdk:fixup-types-imports": "node ./bin/embedding-sdk/fixup-types-after-compilation.js",
     "embedding-sdk:generate-package": "node ./bin/embedding-sdk/generate-sdk-package-files.js",
     "embedding-sdk:publish": "cd ./resources/embedding-sdk && npm publish",
+    "embedding-sdk:test-unit": "yarn test-unit enterprise/frontend/src/embedding-sdk/",
```

Comment:
Just a simple sanity check for SDK, we have more tests when running the uberjar workflow already.